pkg/util/exec/exec.go

```diff
@@ -21,6 +21,9 @@ import (
 	"syscall"
 )
 
+// ErrExecutableNotFound is returned if the executable is not found.
+var ErrExecutableNotFound = osexec.ErrNotFound
```

Comment:
At first this bugged me but now I see it does make the code easier to read (```osexec.ErrNotFound``` isn't very descriptive). :+1: